src/main.ts

```diff
@@ -266,23 +217,13 @@ function main(args: string[]) {
     process.exit(1);
   }
 
-  if (settings.saveTemporaries) {
-    for (let fileName of Object.keys(jsFiles)) {
-      fs.writeFileSync(fileName, jsFiles[fileName]);
-    }
+  for (let fileName of Object.keys(jsFiles)) {
+    fs.writeFileSync(fileName, jsFiles[fileName]);
```

Comment:
Does this always write the files next to the .ts files?  Or does it obey outDir?  Just curious.